src/applications/model/gaming-streaming-server.cc

```diff
@@ -128,26 +138,31 @@ GamingStreamingServer::Send (Ptr<RandomVariableStream> packetSize,
 
   m_socket->Send (packet);
   m_totSentPackets++;
+  m_totSentBytes += pktSize;
+  Time nextTx = MilliSeconds (m_interArrivalTimesVariables[idx]->GetInteger ());
 
   if (Ipv4Address::IsMatchingType (m_peerAddress))
     {
       NS_LOG_INFO ("At time " << Simulator::Now ().GetSeconds ()
                               << "s gaming server sent "
                               <<  packet->GetSize () << " bytes to "
-                              << Ipv4Address::ConvertFrom(m_peerAddress)
-                              << " port " << m_peerPort);
+                              << Ipv4Address::ConvertFrom (m_peerAddress)
+                              << " port " << m_peerPort
+                              << " total sent packets " << m_totSentPackets
+                              << " total Tx " << m_totSentBytes << " bytes");
     }
   else if (Ipv6Address::IsMatchingType (m_peerAddress))
     {
       NS_LOG_INFO ("At time " << Simulator::Now ().GetSeconds ()
                               << "s gaming server sent "
                               <<  packet->GetSize () << " bytes to "
-                              << Ipv6Address::ConvertFrom(m_peerAddress)
-                              << " port " << m_peerPort);
+                              << Ipv6Address::ConvertFrom (m_peerAddress)
+                              << " port " << m_peerPort
+                              << " total sent packets " << m_totSentPackets
+                              << " total Tx " << m_totSentBytes << " bytes");
     }
```

Comment:
```suggestion
if (Ipv4Address::IsMatchingType (m_peerAddress))
  {
    NS_LOG_INFO ("Sending to "<< Ipv4Address::ConvertFrom (m_peerAddress)
                              << " port " << m_peerPort);
  }
else if (Ipv6Address::IsMatchingType (m_peerAddress))
  {
    NS_LOG_INFO ("Sending to "<< Ipv6Address::ConvertFrom (m_peerAddress)
                              << " port " << m_peerPort);
  }
      NS_LOG_INFO ("Sent " <<  packet->GetSize () << " bytes"
                              << ", total sent packets " << m_totSentPackets
                              << ", total Tx " << m_totSentBytes << " bytes");
```

Reply:
In this way, logging information printed in different lines and it's not really interesting.
The better way to avoid the code duplication is  using:

std::stringstream addressString;
      addressString << Ipv4Address::ConvertFrom (m_peerAddress);

Reply:
i don't agree that it is not interesting, you will get 2 subsequent separate lines coming from the same time instant and the same class. doing what you propose would take extra time for the optimized build. I'm not a fan of repeating exactly the same code twice but do as you please, the important thing is the final result.
PS: remember that you can [enable prefixes](https://www.nsnam.org/docs/manual/html/logging.html#prefix-options) to get information on `Simulator::Now`, so you shouldn't add that info yourself